tests/python_package_test/test_dask.py

```diff
@@ -291,6 +265,8 @@ def test_regressor(output, client, listen_port):
     if output != 'dataframe':
         s1 = r2_score(dy, p1)
     p1 = p1.compute()
+    p2_local = dask_regressor.to_local().predict(X)
+    s2_local = dask_regressor.to_local().score(X, y)
```

Comment:
```suggestion
    p1_local = dask_regressor.to_local().predict(X)
    s1_local = dask_regressor.to_local().score(X, y)
```

Please call these `p1_local` and `p2_local`. We have a convention in these tests right now that `p1` references predictions from a model trained with `DaskLGBMRegressor` and the `p2` references things from `lightgbm.sklearn.LGBMRegressor`. I know that's confusing and we should probably change it, but for now this test should be consistent with the existing code.